Makefile

```diff
@@ -8,16 +8,25 @@ ns ?= "test"
 
 # Determine the operating system
 UNAME_S := $(shell uname -s)
+PHP_API_VERSION := $(shell php -i | grep 'PHP API' | awk -F' => ' '{print $$2}')
+PHP_VERSION := $(shell php -v | head -n 1 | awk '{print $$2}' | cut -d. -f1,2)
```

Comment:
Same here. This seems to be a better replacement:
`php -i | grep -Po '(?<=PHP Version => ).*' | uniq`